README.md

```diff
@@ -1,2 +1,215 @@
-# to-do-list-peer-feedback
-Here my coding partner leaves feedback on my project
+<div align="center">
+<h3><b>Zilola - To-Do list</b></h3>
+</div>
+This is a project created in the Microverse curriculum
+
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 "To-Do list" <a name="about-project"></a>
+
+**"To-Do list"** project is a single-page app consisting of the following main files:
+
+- style.css, index.html, index.js files are located in ./src directory;
+- distribution files main.js, index.html are generated by Webpack and served by Webpack dev server from /dist folder.
+
+The repo was documented in a professional way. The Website is simple and dynamic.
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
```

Comment:
- Very Good Read me file which is descriptive  and self explanatory. It has all the required info for downloading and ruining the app.